File: src/main/java/org/example/SortingWithDsa/BubbleSort.java
package org.example.SortingWithDsa;

import java.util.Arrays;

public class BubbleSort {
    public static void main(String[] args) {
        System.out.println("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
        System.out.println("*********** Bubble Sort ****************");
        System.out.println("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");

// In Bubble sort - sorting start from Highest to lowest element in the array.
        int[] numbers = {6,9,7,3,8,5,2,1,4,10};
        for (int i =0; i< numbers.length -1; i++){
            for(int j =0; j <numbers.length -1-i; j++){
                if(numbers[j] >= numbers[j+1]){
                    int temp = numbers[j];
                    numbers[j] = numbers[j+1];
                    numbers[j+1] = temp;
                }
            }
        }
        System.out.println(Arrays.toString(numbers));
    }
}

File: src/main/java/org/example/SortingWithDsa/bubbleSort.js

const bubbleSort=()=>{
    for(let i =0; i <nums.length-1; i++){
        for(let j=0; j<nums.length-1-i; j++){
            if(nums[j]>=nums[j+1]){
                let temp = nums[j];
                nums[j] = nums[j+1];
                nums[j+1] = temp;
            }
        }
    }
    return nums;
}

let nums = [3,1,2,5,4,8,6,7,9];
const result  =bubbleSort(nums);
console.log(result);